src/openms/source/ANALYSIS/OPENSWATH/MRMDecoy.cpp

```diff
@@ -442,28 +471,17 @@ namespace OpenMS
           {
             if (std::fabs(tr.getProductMZ() - decoy_tr.getProductMZ()) < similarity_threshold)
             {
-              if (!exclude_similar)
```

Comment:
If the fragment ions of targets and decoys are too similar, the corresponding decoy peptide will directly be excluded.